web/src/pages/Route/transform.ts

```diff
@@ -158,7 +158,11 @@ export const transformStepData = ({
         default:
           key = `arg_${name}`;
       }
-      return [key, operator, value];
+      let finalValue = value
+      if (operator === "IN") {
+        finalValue = JSON.parse(value as string)
```

Comment:
Does value possible be `""`? It may cause  an error here. `JSON.parse("")`

Reply:
no, for IN operator, users must input Array string.

Reply:
OK